src/platform/FreeBSD.hh

```diff
@@ -60,6 +61,7 @@ private:
 	const LinkerMap linkers_;
 	const std::vector<int> libdirs_;
 	const std::vector<int> pathdirs_;
+	po_map* map;
```

Comment:
The asterisk belongs with the variable name (`po_map *map`) rather than the type name (`po_map* map`). If you were to declare two fields in this way, it would have to look like `po_map *foo, *bar` rather than `po_map* foo, bar` (the latter declares one pointer and one struct value).